.github/workflows/ci.yml

```diff
@@ -20,5 +20,4 @@ jobs:
       - run: yarn install
       - run: yarn ci
         env:
-          CI: true
-      - run: yarn backstage-cli versions:check
```

Comment:
This has no replacement, didn't the Backstage people suggest to replace this with something built into `yarn`?

Reply:
Yes. The replacement is `yarn dedupe`. But this command is only available in yarn-3, and this repo is not using it. So there's no actual replacement unfortunately :( We could only bump yarn here and then use it

Reply:
Ok, then let's try that later, could you comment this line out with a TODO instead so we maybe don't forget to add `yarn dedupe` later?

Reply:
Did that :)